recipes/plantuml/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set version = "1.2021.7" %}
+
+package:
+  name: plantuml
+  version: {{ version }}
+
+source:
+  url: https://github.com/plantuml/plantuml/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: a8e3209171ccc57a9be2a267b2bcea976cc94a535c143ee4a77ba4dab5d71698
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - maven
+    - graphviz
```

Comment:
I don't really know much about Java but I would expect this to be needed in `host` rather than `build`.

Reply:
i'll give it a shot!

Reply:
linux did not like that, will put it back!

Reply:
back to green....